tests/Makefile

```diff
@@ -0,0 +1,59 @@
+SHELL = /bin/sh
```

Comment:
The new `tests` folder contains unit-tests for SBS. They are written with [libcheck](https://github.com/libcheck/check)

There is `README.md`, with short how-to instructions.